server/types/char.go

```diff
@@ -34,6 +34,7 @@ import (
 // BpChar is a char that has an unbounded length. "bpchar" and "char" are the same type, distinguished by the length
 // being bounded or unbounded.
 var BpChar = CharType{Length: stringUnbounded}
+var internalChar = CharType{Length: 1}
```

Comment:
We should add a comment for this type, and we may as well make it a public type too.